### PR TITLE
Fix group name, password, remove tab indentation

### PR DESCRIPTION
**Pull request by viliam-durina**

### CODE REVIEW

**Comment by cangencer** [comment]
Fixes #1565

File: hazelcast-jet-distribution/src/main/resources/bin/cluster.sh [hunk 1]
```diff
@@ -60,21 +60,21 @@ if [ -z "$PORT" ]; then
...
+    PASSWORD="jet-pass"
 fi
 
 if [ -z "$ADDRESS" ]; then
     echo "No specific ip address is defined, running script with default ip : '127.0.0.1'."
     ADDRESS="127.0.0.1"
 fi
 
-command -v curl >/dev/null 2>&1 || { echo >&2 "Cluster state script requires curl but it's not installed. Aborting."; exit -1; }
+command -v curl >/dev/null 2>&1 || { echo >&2 "Cluster state script requires curl but it's not installed. Aborting."; exit 255; }
```
Comment (cangencer) [inline hazelcast-jet-distribution/src/main/resources/bin/cluster.sh]: why not `1`?

Comment (viliam-durina) [reply hazelcast-jet-distribution/src/main/resources/bin/cluster.sh]: `shellchecker` told me that -1 isn't valid. Why not 255 then? :)

Comment (cangencer) [reply hazelcast-jet-distribution/src/main/resources/bin/cluster.sh]: seems like a non-standard error. 1 is typically used a a general error exit code. `255` means "exit status out of range"

**Comment by cangencer** [comment]
verify

**Pull request merged**